doc/advanced_configuration.rst

```diff
@@ -426,18 +427,24 @@ dictionary following the pattern below::
       'binder': {
          'org': '<github_org>',
          'repo': '<github_repo>',
-         'url': '<binder_url>',  # URL serving binders (e.g. mybinder.org)
-         'branch': '<repo_branch>',  # Can also be a tag or commit hash
-         'dependencies': '<list_of_paths_to_dependency_files>'
+         'url': '<binder_url>',  # Any URL of a binder server. Must be full URL (e.g. https://mybinder.org).
+         'branch': '<branch-for-documentation>',  # Can be any branch, tag, or commit hash. Use a branch that hosts your docs.
+         'dependencies': '<list_of_paths_to_dependency_files>',
+         'filepath_prefix': '<prefix>' # Optional, a prefix to append to any filepaths in Binder links.
+                            use this if you move your built documentation to a sub-folder of your repository (e.g., "v2.1")
          }
     }
 
+Note that ``branch:`` should be the branch on which your documentation is hosted.
+If you host your documentation on GitHub, this is usually ``gh-pages``.
```

Comment:
or master? I think we are using master for scikit-learn but maybe we are the only ones doing that?

Reply:
Sphinx-Gallery is also on master as we use a separate repository to host the generated html docs